elements/pfe-content-set/README.md

```diff
@@ -27,6 +27,117 @@ Each header must have an attribute of `pfe-content-set--header` and each panel m
 
 ```
 
+## Attributes
+
+**`pfe-tab-history`** (observed)
+
+If `pfe-content-set` renders as `pfe-tabs`, the `pfe-tab-history` attribute
+enables the component to update window.history and the URL to create sharable
+links.
+
+With the `pfe-tab-history` attribute, `pfe-content-set` and elements with the
+`pfe-content-set--header` attribute *must* have an `id` attribute set for this
+to work.
+
+##### Example Markup
+```html
+<pfe-content-set id="my-content-set" pfe-tab-history>
+  <h2 pfe-content-set--header id="heading1">Heading 1</h2>
+  <p pfe-content-set--panel id="panel1">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore </p>
+</pfe-content-set>
+```
+
+becomes
+
+```html
+<pfe-content-set id="my-content-set" pfe-tab-history>
+  <pfe-tabs pfe-id="my-content-set">
+    <pfe-tab pfe-id="heading1">
+      <h2 pfe-content-set--header id="heading1">Heading 1</h2>
+    </pfe-tab>
+    <pfe-tab-panel pfe-id="panel1">
+      <p pfe-content-set--panel id="panel1">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore </p>
+    </pfe-tab-panel>
+  </pfe-tabs>
+</pfe-content-set>
+```
+Note how the `id` attributes from `pfe-content-set` and elements with the
+`pfe-content-set--header` attribute pass the value of the `id` attribute to
+its corresponding tab element and sets the `pfe-id` attribute.
+
+#### URL Pattern
+
+The URL pattern will be `?{id-of-tabs}={id-of-selected-tab}`. In the example
+below, selecting "Heading 2" will update the URL as follows:
+`?my-content-set=heading2`.
```

Comment:
one one possible suggestion, you could add that this can be used in conjunction with hashes in the URL, to jump to that band and open a particular tab. 

`?my-content-set=header-3#with-tab-history`